### ISSUE TITLE
Create ByteBuffer from existing ArrayBuffer?

### ISSUE THREAD

**Issue by stodge** [protobufjs/bytebuffer.js#98]
This is probably a silly question, but how can I initialise a ByteBuffer instance from an existing ArrayBuffer? Thanks

**Comment by dcodeIO** [comment]
Seems you are looking for: https://github.com/protobufjs/bytebuffer.js/wiki/API#bytebufferwrapbuffer-encoding-littleendian-noassert

**Comment by stodge** [comment]
Thanks - I tried that but I couldn't get it to work. Let me try again.

For context, I'm trying to parse binary data from a websocket.

**Comment by stodge** [comment]
I tried this:

```
a = new Uint8Array([0,1,2,3,4,5,6,7])
b = new dcodeIO.ByteBuffer(a.buffer)
b.readByte()
bytebuffer.js:568 Uncaught RangeError: Illegal offset: 0 <= 0 (+1) <= 0

b = new dcodeIO.ByteBuffer(a)
b.readByte()
bytebuffer.js:568 Uncaught RangeError: Illegal offset: 0 <= 0 (+1) <= 0
```


I'm just starting out with working with binary data in JS, so my understanding may be lacking.

Thanks
